src/test/java/DukeTest.java

```diff
@@ -0,0 +1,106 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.siawsam.duke.Deadline;
+import com.siawsam.duke.DukeException;
+import com.siawsam.duke.Event;
+import com.siawsam.duke.Response;
+import com.siawsam.duke.Task;
+import com.siawsam.duke.TaskList;
+import com.siawsam.duke.TaskSearcher;
+import com.siawsam.duke.Todo;
+import com.siawsam.duke.Ui;
+
+public class DukeTest {
```

Comment:
I think it will be better if you could split the unit tests into individual test classes for each class you have.